jedi.inc

```diff
@@ -639,7 +639,10 @@
 { Set FreePascal to Delphi mode }
 {$IFDEF FPC}
   {$MODE DELPHI}
-  {$ASMMODE Intel}
+  // Assembler mode Intel is only available for the x86 CPU family
+  {$IF DEFINED(CPUI8086) OR DEFINED(CPUI386) OR DEFINED(CPUX86_64)}
+    {$ASMMODE Intel}
+  {$ENDIF}
```

Comment:
Please use `{$IFEND}` instead, for Delphi 7 compatibility

Reply:
Ah, right, sorry. I forgot that older Delphi versions didn't support that (in addition to Delphi's preprocessor not ignoring disabled code) 🙄😅

Reply:
Same here... I pulled your change into my projects earlier and was using it happily until today when I tried compiling one of them in D7, more or less by accident. 😄 Thanks!

Reply:
Then it's a good thing you tested with an older version. 😉